Remove modules that require software not yet ported to Solaris, such as
Network Manager.

Even if we ported all their dependencies, it's not clear we want these modules,
since they are focused on setting up a new machine, not a new user - we have
our initial user creation & network configuration handled by our OS install
software instead.

Not appropriate for upstream.

--- gnome-initial-setup-3.18.0/configure.ac	Mon Nov 30 20:11:09 2015
+++ gnome-initial-setup-3.18.0/configure.ac	Mon Nov 30 20:12:09 2015
@@ -29,10 +29,6 @@
 GEOCLUE_REQUIRED_VERSION=2.1.2
 
 PKG_CHECK_MODULES(INITIAL_SETUP,
-                  NetworkManager >= $NETWORK_MANAGER_REQUIRED_VERSION
-                  libnm-glib >= $NETWORK_MANAGER_REQUIRED_VERSION
-                  libnm-util >= $NETWORK_MANAGER_REQUIRED_VERSION
-                  libnm-gtk >= $NETWORK_MANAGER_REQUIRED_VERSION
                   polkit-gobject-1 >= $POLKIT_REQUIRED_VERSION
                   accountsservice
                   gnome-desktop-3.0 >= $GNOME_DESKTOP_REQUIRED_VERSION
@@ -48,7 +44,6 @@
                   rest-0.7
                   json-glib-1.0
 		  libsecret-1
-                  pwquality
                   webkit2gtk-4.0)
 
 GEOCLUE_DBUS_INTERFACE_XML=`pkg-config --variable=dbus_interface geoclue-2.0`

--- gnome-initial-setup-3.18.0/gnome-initial-setup/pages/Makefile.am	Tue Dec  1 17:41:00 2015
+++ gnome-initial-setup-3.18.0/gnome-initial-setup/pages/Makefile.am	Tue Dec  1 17:41:06 2015
@@ -4,10 +4,7 @@
 	region \
 	keyboard \
 	eulas \
-	network \
 	timezone \
 	privacy \
 	goa \
-	account \
-	password \
 	summary
--- gnome-initial-setup-3.18.0/gnome-initial-setup/Makefile.am	Tue Dec  1 19:06:50 2015
+++ gnome-initial-setup-3.18.0/gnome-initial-setup/Makefile.am	Tue Dec  1 19:05:01 2015
@@ -35,12 +35,9 @@
 	pages/region/libgisregion.la \
 	pages/keyboard/libgiskeyboard.la \
 	pages/eulas/libgiseulas.la \
-	pages/network/libgisnetwork.la \
 	pages/timezone/libgistimezone.la \
 	pages/privacy/libgisprivacy.la \
 	pages/goa/libgisgoa.la \
-	pages/account/libgisaccount.la \
-	pages/password/libgispassword.la \
 	pages/summary/libgissummary.la \
 	$(INITIAL_SETUP_LIBS) \
 	-lm
--- gnome-initial-setup-3.18.0/gnome-initial-setup/gnome-initial-setup.c	Wed Dec  2 09:17:11 2015
+++ gnome-initial-setup-3.18.0/gnome-initial-setup/gnome-initial-setup.c	Wed Dec  2 09:17:29 2015
@@ -65,12 +65,9 @@
   /* PAGE (region,   FALSE), */
   PAGE (keyboard, FALSE),
   PAGE (eula,     FALSE),
-  PAGE (network,  FALSE),
   PAGE (privacy,  FALSE),
   PAGE (timezone, TRUE),
   PAGE (goa,      FALSE),
-  PAGE (account,  TRUE),
-  PAGE (password, TRUE),
   PAGE (summary,  FALSE),
   { NULL },
 };
